client/src/components/pages/NeowisePage.js

```diff
@@ -34,6 +35,10 @@ function NeowisePage() {
     setNasaItems(nasa);
   };
 
+  useEffect(() => {
+    setTimeout(() => setLazyButton(true), 2500);
+  });
```

Comment:
Technically, it's possible that you change the page during the timeout.
This would result in a "set state of unmounted component" warning.

Thus, you have to clear the timeout, when you unmount the component:
In addition, you create a new timeout on every render. Don't forget to add effect dependencies (`[]`);

```js
  useEffect(() => {
    const timeoutId = setTimeout(() => setLazyButton(true), 2500);
    return () => {
      clearTimeout(timeoutId);
    } 
  }, []);
```

Reply:
Thanks for review! 😊